packages/sanity/src/core/form/studio/tree-editing/components/search/TreeEditingSearchMenu.tsx

```diff
@@ -0,0 +1,67 @@
+import {Box, Button, Stack, Text} from '@sanity/ui'
+import {isEqual} from 'lodash'
+import {useCallback} from 'react'
+import {CommandList, type Path, supportsTouch} from 'sanity'
+
+import {type TreeEditingMenuItem} from '../../types'
+import {ITEM_HEIGHT} from './constants'
+
+interface TreeEditingSearchMenuProps {
+  items: TreeEditingMenuItem[]
+  onPathSelect: (path: Path) => void
+  textInputElement: HTMLInputElement | null
+}
+
+export function TreeEditingSearchMenu(props: TreeEditingSearchMenuProps): JSX.Element {
+  const {items, onPathSelect, textInputElement} = props
+
+  const getItemDisabled = useCallback((index: number) => false, [])
```

Comment:
Unsure why the code on [TreeEditingBreadcrumbsMenu.tsx](https://github.com/sanity-io/sanity/pull/6804/files#diff-b7e3ae4e075794063b24f74042a8fd13fef1afc913d42a28a994dda97ec34b6d) was changed to 

```
  const getItemDisabled = useCallback(
    (index: number) => {
      const item = items[index]
      return isEqual(item.path, selectedPath)
    },
    [items, selectedPath],
  )
```

and this remained the same? 🤔

Reply:
There's currently no concept of a "selected state" in the search menu so we always keep the items enabled

Reply:
Ah I see, I would likely add this comment to this part of the code, just to have in the future, but I'm also ok with leaving it as is then :)

Reply:
I'll remove the function entirely since it is not actually doing anything :)